app/src/engineering/java/com/hedvig/app/feature/tracking/Module.kt

```diff
@@ -1,12 +1,15 @@
+@file:Suppress("RemoveExplicitTypeArguments")
+
 package com.hedvig.app.feature.tracking
 
+import com.hedvig.app.feature.hanalytics.HAnalyticsSink
 import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.dsl.bind
 import org.koin.dsl.module
 
 val trackingLogModule = module {
     viewModel { TrackingLogViewModel(get()) }
-    single { EngineeringTracker() } bind HAnalyticsSink::class
-    single { DebugLogTrackerSink() } bind HAnalyticsSink::class
-    single { ShakeTrackerSink() } bind HAnalyticsSink::class
+    single<EngineeringTrackerSink> { EngineeringTrackerSink() } bind HAnalyticsSink::class
+    single<DebugLogTrackerSink> { DebugLogTrackerSink() } bind HAnalyticsSink::class
+    single<ShakeTrackerSink> { ShakeTrackerSink() } bind HAnalyticsSink::class
```

Comment:
Why do we need explicit type arguments here? 🤔

Reply:
Yeah we don't *need* them, but I've grown to really like being explicit with Koin and what kind of types the `singles` provide.
I remember talking about this before here https://github.com/HedvigInsurance/android/pull/891#discussion_r803653451 and I feel like if we did an explicit paramter for every single thing we provide with Koin it would make it easier for our eyes. Maybe you didn't see that comment though, because I haven't heard your opinion on it :D

Reply:
Fair enough 🙂